x-pack/plugins/security_solution/server/usage/detections/detections.mocks.ts

```diff
@@ -302,3 +302,179 @@ export const getMockMlDatafeedStatsResponse = () => ({
     },
   ],
 });
+
+export const getMockRuleSearchResponse = (immutableTag: string = '__internal_immutable:true') => ({
+  took: 2,
+  timed_out: false,
+  _shards: {
+    total: 1,
+    successful: 1,
+    skipped: 0,
+    failed: 0,
+  },
+  hits: {
+    total: {
+      value: 1093,
+      relation: 'eq',
+    },
+    max_score: 0,
+    hits: [
+      {
+        _index: '.kibanaindex',
+        _id: 'alert:6eecd8c2-8bfb-11eb-afbe-1b7a66309c6d',
+        _score: 0,
+        _source: {
+          alert: {
+            name: 'Azure Diagnostic Settings Deletion',
+            tags: [
+              'Elastic',
+              'Cloud',
+              'Azure',
+              'Continuous Monitoring',
+              'SecOps',
+              'Monitoring',
+              '__internal_rule_id:5370d4cd-2bb3-4d71-abf5-1e1d0ff5a2de',
+              `${immutableTag}`,
```

Comment:
nit: unnecessary interpolation
```suggestion
              immutableTag,
```